internal/compiler/wasm/wasm.go

```diff
@@ -463,7 +510,7 @@ func (c *Compiler) compileExternalFuncDecls() error {
 		},
 	}
 
-	return c.emitFunction("builtins", c.code)
+	return c.storeFunc("builtins", c.code)
```

Comment:
This has little to do with the wasm-opt integration; I was in the middle of adding optimization scaffolding when I decide to give wasm-opt a go 😄